examples/DataStax/util.js

```diff
@@ -0,0 +1,8 @@
+function getClientArgs() {
+    return {
+        contactPoints: [process.env.SCYLLA_URI ?? '172.17.0.2'],
```

Comment:
last nit: let's add port here, too, so that a user gets a correct hint that it's expected to be passed.
```suggestion
        contactPoints: [process.env.SCYLLA_URI ?? '172.17.0.2:9042'],
```